frontend/src/components/episode/episode.tsx

```diff
@@ -120,6 +121,17 @@ const Episode: React.FC = () => {
     return <Loader />;
   }
 
+  const handleClickStartLive = (): void => {
+    if (episode) {
+      dispatch(episodeActions.changeEpisodeStatus({
+        ...episode,
+        status: EpisodeStatus.LIVE,
```

Comment:
according to the task: 
`Live` status can be selected only on the live page (by the button) "Go to Live"
**on the live page**